test/powershell/Modules/Microsoft.PowerShell.Utility/Set-Date.Tests.ps1

```diff
@@ -1,12 +1,12 @@
-Describe "Set-Date for admin" -Tag @('CI', 'RequireAdminOnWindows') {
+Describe "Set-Date for admin" -Tag @('CI', 'RequireAdminOnWindows', 'RequireSudo') {
```

Comment:
Maybe `'RequireAdminOnUnix'`?

Reply:
I think it should be `RequireSudo`.  It could be `RequireSudoOnUnix`